contrib/90-sssd-token-access.rules.in

```diff
@@ -0,0 +1,7 @@
+# this udev file should be used with udev 188 and newer
+ACTION!="add|change", GOTO="sssd_end"
+
+# Allow SSSD to access security tokens
+SUBSYSTEM=="hidraw", ENV{ID_SECURITY_TOKEN}=="1", RUN{program}+="/usr/bin/setfacl -m g:@SSSD_USER@:rw $env{DEVNAME}"
```

Comment:
Btw, why additional group and not user access?
Can't think of any practical difference, so just wondering.